SKSpriteButton/Classes/SKSpriteButton.swift

```diff
@@ -47,38 +47,116 @@ public class SKSpriteButton: SKSpriteNode {
         case reentry
     }
     
+    // Button becomes a toggle switch that switches between normal and tapped
+    public var toggleMode: Bool = false
```

Comment:
nitpicking: isToggleMode seems more consistent with other naming in the code base.

Reply:
I agree